examples/destination/python/destination/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+python3 -m venv destination_run
+source destination_run/bin/activate
+mkdir -p protos
+cp ../../../../*.proto protos/
+pip install -r requirements.txt
+python -m grpc_tools.protoc \
```

Comment:
I have done changes in the python connector example PR to include [generated files in a folder](https://github.com/fivetran/fivetran_sdk/pull/29/commits/f8eba6a22047c5efbf1ea77c205f0c9112a97255#diff-6bc98c7ec70f37c6c988c921ae93c423aff731e1d4416c0981d83d5bac9e47eeR10-R12), and then [import it](https://github.com/fivetran/fivetran_sdk/pull/29/commits/f8eba6a22047c5efbf1ea77c205f0c9112a97255#diff-5ef619cbb418662511c9562196ae2c77c7397443e8dc447afa766ab96b9102dfR5-R6).